docs/content/preview/architecture/transactions/concurrency-control.md

```diff
@@ -0,0 +1,1201 @@
+---
+title: Concurrency control
+headerTitle: Concurrency control
+linkTitle: Concurrency control
+description: Details about Concurrency control in YSQL
+menu:
+  preview:
+    identifier: architecture-concurrency-control
+    parent: architecture-acid-transactions
+    weight: 1153
+type: docs
+---
+
+[Concurrency control](https://en.wikipedia.org/wiki/Concurrency_control) in databases ensures that multiple transactions can execute concurrently while preserving data integrity. Concurrency control is essential for correctness in environments where two or more transactions can access the same data at the same time.
+
+YugabyteDB provides two policies to handle conflicts between concurrent transactions as described in the following sections. 
+
+For information on how row-level explicit locking clauses behave with these concurrency control policies, refer to [Row-level explicit locking clauses](#row-level-explicit-locking-clauses) .
+
+## Fail-on-Conflict
+
+This is the default concurrency control strategy and is applicable for `REPEATABLE READ` and `SERIALIZABLE` isolation levels.
+It is not applicable for [Read Committed](../read-committed/) isolation.
+
+In this mode, transactions are assigned random priorities with some exceptions as described in [Transaction Priorities](../transaction-priorities/).
+If a conflict occurs, a transaction with the lower priority is aborted. There are two possibilities when a transaction T1 tries to read or write or lock a row in a mode conflicting with a other concurrent transactions as follows:
```

Comment:
done, also kept "other ... transactions" instead of "another ... transaction" since the latter doesn't make sense with the explanations of `wound` and `die` that follow